meta/sai_rpc_frontend.cpp

```diff
@@ -0,0 +1,870 @@
+/**
+ * Copyright (c) 2021 Microsoft Open Technologies, Inc.
```

Comment:
can this file be located in ptf directory ?

Reply:
This file make use of autogenerated RPC code which also is to be placed in meta directory and its subdirectories. Please see: https://github.com/opencomputeproject/SAI/pull/1325/files

Reply:
We can maybe allow the gensairpc script to allow to generate the files in a different directory than meta as an argument. Would it help if we create a top-level "thriftrpc" directory to place this and other related files?

Reply:
@kcudnik - please confirm?

Reply:
what should i confirm here?

Reply:
@kcudnik  - wanted to make sure if we're in agreement on "create a top-level "thriftrpc" directory to place this and other related files"

Reply:
that should be fine